src/Windows/Avalonia.Win32/Input/Imm32InputMethod.cs

```diff
@@ -132,6 +132,13 @@ public void SetCursorRect(Rect rect)
                         ptCurrentPos = new POINT { X = x1, Y = y1 },
                     };
                     ImmSetCompositionWindow(himc, ref compForm);
+
+                    var logFont = new LOGFONT()
+                    {
+                        lfHeight = y2-y1,
+                        lfQuality = 5 //CLEARTYPE_QUALITY
+                    };
+                    ImmSetCompositionFont(himc, ref logFont);
```

Comment:
cool!

Reply:
I've also tried hard to get a current font family name, is bold, is italic, is underlined from focused textBox and apply here, but I think @kekekeks will be not so happy about this because it adds a lot of code, so I've decided to provide a lot cleaner solution, and f it's ok - then I will push another PR.

Reply:
We could _optionally_ provide the current font from ITextInputClient, but that would require also invalidation events and support from the text box. That would also be a breaking change, so it get the feature into 0.10.x we'd need a separate `ITextInputClientWithFontInformation` interface.